src/public-lib/components/WindowWrapper.ts

```diff
@@ -0,0 +1,33 @@
+import autobind from "autobind-decorator";
```

Comment:
Moved from `shared/components/window/WindowStore.ts` (to support responsive lollipop plot in `react-mutation-mapper`)